.github/workflows/ci.yml

```diff
@@ -22,4 +22,4 @@ jobs:
             export DISPLAY=:99.0
             Xvfb $DISPLAY &
           fi
-          mvn -V -B -e verify -Pe4.12 -Dtycho.executionEnvironment=JavaSE-17 -DskipUITests=false
+          mvn -V -B -e verify -Pe4.12 -Dtycho.executionEnvironment=JavaSE-17 -DskipUITests=false -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
```

Comment:
@marchof Nice! Just one question - did you choose
```
-Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
```
over
```
--no-transfer-progress
```
to be able to see warnings?

Reply:
Now that you mentioning it I remember this is the better option (less verbose, more explicit). I will change it.

Reply:
Fixed.